buildozer/targets/osx.py

```diff
@@ -62,9 +62,9 @@ def download_kivy(self, cwd):
                     sys.exit(1)
 
             self.logger.info('Extracting and installing Kivy...')
-            check_call(('hdiutil', 'attach', cwd + '/Kivy.dmg'))
-            buildops.file_copy(
-                '/Volumes/Kivy/Kivy.app', './Kivy.app', cwd=cwd)
+            check_call(('hdiutil', 'attach', f'{cwd}/Kivy.dmg'))
```

Comment:
It would be nice if this used PathLib paths, but I acknowledge this will only run on OSX so this is purely aesthetic, not important.

Reply:
It using now, I think you reviewed outdated code. At least this line is.